CMakeLists.txt

```diff
@@ -12,25 +12,8 @@ find_package(Boost REQUIRED system filesystem)
 find_package(ASSIMP QUIET)
 if (NOT ASSIMP_FOUND)
   find_package(PkgConfig REQUIRED)
-  pkg_check_modules(ASSIMP assimp)
-endif()
-if (ASSIMP_FOUND)
```

Comment:
> it is officially unsupported for two years now and osrf stopped building debs for it.

Did not realize that, I thought the Indigo LTS applied to all supported Ubuntu versions

> Many low-level packages were still missing in kinetic last time I checked

MoveIt is now working on Kinetic with only two missing dependencies that are not in the ``ros-planning`` org - manipulation_msgs and household_objects_database_msgs. I think many groups are not using kinetic yet because projects like MoveIt! are not released into it yet. That's our job ;-)

> I tend to apply all ABI stable patches to indigo before moving them

That makes sense. These changes just seem risky given the complexity of multi-Ubuntu distro dependency chains. I'm of the opinion we should strongly encourage all non-bug fix changes go into the latest branches.